packages/openapi-generator/src/parser/schema.ts

```diff
@@ -59,13 +51,28 @@ export function parseSchema(
   }
 
   if (schema.allOf?.length) {
+    if (schema.properties) {
```

Comment:
[req] I think this needs to be handled for anyOf and oneOf as well.

Reply:
Sorry I had somehow missed this again. I moved the logic to the `parseXOfSchema` function now.